src/org/usfirst/frc/team2399/robot/commands/AgitatorBackward.java

```diff
@@ -0,0 +1,44 @@
+package org.usfirst.frc.team2399.robot.commands;
+
+import org.usfirst.frc.team2399.robot.Robot;
+import org.usfirst.frc.team2399.robot.RobotMap;
+import org.usfirst.frc.team2399.robot.subsystems.Agitator;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ * Moves agitator motor backwards
+ */
+
+public class AgitatorBackward extends Command {
```

Comment:
Would it be possible to consolidate a bunch of this agitator code into a single parent class that gets constructed with different speeds to set?